### DISCUSSION TITLE
Quantum of Fees

### DISCUSSION THREAD

**Discussion by Aaron-XReg** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#40] [General]
The fees proposed for trading platforms and custodians in the VASP Regulations may be prohibitive for certain prospective market participants. 

In particular the $100,000 fee for trading platforms with revenue not exceeding ten million dollars could be a barrier to market access for new trading platforms. I would recommend reducing the fee to $70,000 and creating a new category for firms that generate revenues of under two million dollars. As revenues increase the higher fees would apply but this would be more reasonable for firms that are at an earlier stage of growth.

With respect to custodians, the fees are bifurcated between firms with revenue above and below one million in revenue (60k and 30k respectively). I would recommend increasing the 1 million dollar threshold to at least five million dollars in revenue.

**Comment by AGIAI** [comment]
For firms generating under $2m of revenue; there should be some oversight (self-regulatory organization" (SRO) model) but nominal fees (c. $3,000-$4,000; which would be inline with the SIBA requirements. 

This should apply for custody, trading and issuance.

Cayman should be encouraging innovation with guardrails; no pricing it out off market.

**Comment by FinalTwist** [comment]
How do the current proposed fees for trading platform compare with other jurisdictions?  Hong Kong, Singapore, Bermuda, BVI, etc?

**Reply by Aaron-XReg** [reply]
The fees for a digital asset exchange in Bermuda are the highest in the Caribbean. The Bermuda Monetary Authority (BMA) annual licensing fee for an exchange is the higher of US$150,000 and 0.00075 of client receipts up to a maximum of US$450,000. Bermuda is a popular destination for digital asset exchanges and they are willing to incur these fees because the BMA has a strong reputation and high service standards (application processing times, responsiveness, etc.). 

Digital asset derivatives have also been included in their DAB Act which is critical for most exchanges (see discussion thread on VA derivatives.)

**Reply by Aaron-XReg** [reply]
The Fees in BVI are quite a bit lower. 15k for custodians and 25k for exchanges. I would not be surprised if these are raised at some point.

**Reply by AGIAI** [reply]
The fees in Dubai which is becoming the leading jurisdiction for the sector are nill;
Abu Dhabi are around 15k;

**Comment by Aaron-XReg** [comment]
The fees for an exchange in Bahamas are also 15k.

**Comment by Aaron-XReg** [comment]
If Cayman is going to charge high end fees like Bermuda it will be necessary to enhance the VASP Act and have very high service standards for applicants. The Bermuda government also does a lot to promote the digital asset industry to attract major firms to the jurisdiction. 

Cayman has the foundations to be a global leader in the VA space, generate significant fee revenue and stimulate innovative economic growth but currently our competitor jurisdictions are doing more to attract leading virtual asset firms. I am confident that with some enhancements to the VASP Act and an improved authorization process we can quickly make up for lost opportunities.

**Comment by Cayman-Blockahain-Association** [comment]
Based on inputs form our members the general consensus was that fees are not competitive compared to other jurisdiction.  The initial fee ranges suggested by Aaron seem reasonable and could potentially be lower when compared to other jurisdictional rates highlighted above. 

An additional consideration suggested by our members would be to base fees on revenue to protect startups and incentivize them to pick the jurisdiction and fees increase as the entity grows.
